src/main/java/org/opentripplanner/graph_builder/module/PruneFloatingIslands.java

```diff
@@ -40,8 +39,6 @@
      */
     private String islandLogFile;
 
-    private StreetLinkerModule transitToStreetNetwork;
```

Comment:
This variable was never instantiated, so I deleted it.

Reply:
I think it's good to clean this out, in fact the whole PruneFloatingIslands needs some cleanup and simplification.